### PR TITLE
[BB-3701] Implement Logo Update Screen page

### PR DESCRIPTION
**Pull request by spokerman12**
We're fulfilling the following user story:

> "As an OCIM user, I want better-looking Logo update screen"

**References**: [BB-3701](https://tasks.opencraft.com/browse/BB-3701), [Gitlab issue](https://gitlab.com/opencraft/dev/opencraft/-/issues/702)

**Screenshots**: See Gitlab issue

**Merge deadline**: "None" 

**Testing instructions**:

1. Pull this branch to your local OCIM environment.
2. Log in with any account that has access to the console, check the Logo component on the sidebar
3. If you don't have an account set up, put a return on top of `validate_available_subdomain` in `registration/models.py`, and a dummy string in lmsUrl = props.activeInstance.data.lmsUrl; on` frontend/src/ui/components/Header/Header.tsx`. That should be enough for you to create an account.

**Author notes and concerns**:

1. This component is present on the Hero edit menu, but this was tested and works as one would expect. Styling might be a concern but that should be tackled further down the line. 

**Reviewers**
- [ ] @farhaanbukhsh

### CODE REVIEW

File: frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx [hunk 1]
```diff
@@ -20,6 +24,10 @@ export const ImageUploadField: React.FC<ImageUploadFieldProps> = (
   const [show, setShow] = React.useState(false);
   const [image, setImage] = React.useState();
 
+  let file: any = '';
+  if (image !== undefined) {
+    file = image;
```
Comment (spokerman12) [inline frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx:29]: This is very ugly but it's the cleanest way I could make Typescript happy. Suggestions are welcome

Comment (farhaanbukhsh) [reply frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx:29]: I have a better suggestion, you can define the type of image here.

```
interface Image {
  name?: string
}
```

then you can use `{image !== undefined ? (<h3>{image.name}</h3>): ''}`

**Review by farhaanbukhsh — CHANGES_REQUESTED** [review]
I have added few comments it looks good overall. Can we just fix the alert box situation and this is gold. :medal_sports:

File: frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx (hunk 1 above)
Comment (spokerman12) [reply frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx:29]: Ok, I added something based on that. Thank you!

File: frontend/src/console/components/Hero/Hero.tsx [hunk 2]
```diff
@@ -359,7 +359,8 @@ export class HeroComponent extends React.PureComponent<Props, State> {
                         clearError={() => {
                           this.props.clearErrorMessage('heroCover');
                         }}
-                        recommendedSize="1200x250px"
+                        parentMessages={messages}
```
[the remark this answers is not in the document]
Comment (spokerman12) [reply frontend/src/console/components/Hero/Hero.tsx:362]: But ImageUploadField has its own messages, separate from the recommendations/warnings, which depend on the parent's messages. That's the reasoning behind this prop.

File: frontend/src/ui/components/ImageUploadField/styles.scss [hunk 3]
```diff
@@ -16,9 +16,9 @@
         color: #333333;
         margin-bottom: 5px;
     }
-    .error-box {
-        margin-top: 20px;
-    }
+    // .error-box {
+    //     margin-top: 20px;
+    // }
```
Comment (farhaanbukhsh) [inline frontend/src/ui/components/ImageUploadField/styles.scss:21]: Please remove the comments

File: frontend/src/console/components/Hero/Hero.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (farhaanbukhsh) [reply frontend/src/console/components/Hero/Hero.tsx:362]: makes sense, my bad

**Review by farhaanbukhsh — APPROVED** [review]
👍 

- [x] I tested this: I have tested this locally, styles and components are fine.
- [x] I read through the code
- [ ] ~~I checked for accessibility issues~~
- [ ] ~~Includes documentation~~
- [ ] ~~I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.~~

File: frontend/src/ui/components/ImageUploadField/styles.scss (hunk 3 above)
Comment (spokerman12) [reply frontend/src/ui/components/ImageUploadField/styles.scss:21]: Removed!

**Pull request merged**